.github/actions/setup-github-action-host/action.yml

```diff
@@ -0,0 +1,69 @@
+name: Setup GitHub Action Host Workflow
```

Comment:
Can we name this exactly for what it is such as setup-integration-test? For example, tasks like build and run unit tests don't need this (or all of this).

Reply:
Yeah that sounds good.